src/main/scala/cromwell/binding/package.scala

```diff
@@ -30,5 +30,9 @@ package object binding {
 
   implicit class EnhancedFullyQualifiedName(val fqn: FullyQualifiedName) extends AnyVal {
     def isScatter = fqn.contains(Scatter.FQNIdentifier)
+    def scopeAndVariableName: (String, String) = {
```

Comment:
While I applaud the move to here, perhaps now would be the time to let this concept graduate to a proper type?